cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -269,20 +270,21 @@ def _add_external_slurmdbd_launch_template(self):
             type="String",
             description="The SSH key name to access the instance (for management purposes only)",
         )
-
         private_ip = CfnParameter(
             self,
             "PrivateIp",
             type="String",
             description="Static private IP address + prefix to assign to the slurmdbd instance",
         )
-
         private_prefix = CfnParameter(
             self,
             "PrivatePrefix",
             type="String",
             description="Subnet prefix to assign with the private IP to the slurmdbd instance",
         )
+        dbms_client_sg_id = CfnParameter(
```

Comment:
NIT: We should have a central function (probably in the constructor) to load all the CfnParameters we need for this stack.
It's difficult to trace all the `CfnParameters` associated with the stack when some are defined in the constructor, and others in lower depth functions.

Reply:
I'm not yet fully sure about this. On one side I agree with you, on the other we can make it clearer which part of the stack the `CfnParameters` are used for.

I will create an internal task to deal with this later in the project.